core/field_angle.js

```diff
@@ -295,6 +307,7 @@ Blockly.FieldAngle.prototype.updateGraph_ = function() {
   }
   // Always display the input (i.e. getText) even if it is invalid.
   var angleDegrees = Number(this.getText()) + Blockly.FieldAngle.OFFSET;
+  angleDegrees = angleDegrees % 360;
```

Comment:
This is a rare opportunity to use the `%=` operator.
```
angleDegrees %= 360;
```

While you're at it, there's another opportunity for a `%=` in `Blockly.FieldAngle.prototype.doClassValidation_`.

Reply:
Oooo fun! :D

Fixed